extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php

```diff
@@ -1,17 +1,19 @@
 <?php
 
 /**
- * Script that prepares XML dumps with the latest revisions of starter wikis
+ * Script that prepares XML and SQL dumps with the latest revisions and *links tables rows of starter wikis
  *
- * Generate XML dumps for all starter wikis and upload them to DFS
+ * Generate XML and SQL dumps for all starter wikis and upload them to DFS
+ *
+ * s3cmd -c /etc/s3cmd/sjc_prod.cfg ls s3://starter/dumps/
  *
  * @see PLATFORM-1305
  *
  * @author Macbre
  * @ingroup Maintenance
  */
 
-putenv('SERVER_ID=80433'); // run in the context of www.wikia.com (where CreateWiki is enabled)
+putenv( 'SERVER_ID=80433' ); // run in the context of www.wikia.com (where CreateWiki is enabled)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)